### ISSUE TITLE
Enhancement: 

### ISSUE THREAD

**Issue by Fixthebug307** [ninjamuffin99/Funkin#1143]
Released the source code of week 7 please :D

**Comment by VMGuy23** [comment]
They're working on cutscenes for desktop
have patience

**Comment by nickpythonluajava** [comment]
its somewhere on github

**Comment by VMGuy23** [comment]
That is the **compiled HTML5 version**. The source does not exist yet.

**Comment by nickpythonluajava** [comment]
> That is the **compiled HTML5 version**. The source does not exist yet.

i mean it does contain the assets

**Comment by VMGuy23** [comment]
Yeah true but that's not the source. The compiled version must contain assets in their own folder due to how HaxeFlixel works, and cutscenes do not work on desktop (can't be compiled)

**Comment by jbmagination** [comment]
@nickpythonluajava mine, as has already been stated, is just the compiled HTML5 version. no source available

**Comment by bonkmaykrQ** [comment]
It's not finished. Just be patient.

They want to keep the release a surprise.

**Comment by GOOBOZA** [comment]
> They're working on cutscenes for desktop
> have patience

thats not why it is taking a long they're making it with week 8
